projects/core/src/product/store/actions/product.action.ts

```diff
@@ -38,5 +40,16 @@ export class LoadProductSuccess extends EntityScopedLoaderActions.EntityScopedSu
   }
 }
 
+export class ClearProductPrice extends EntityScopedLoaderActions.EntityScopedResetAction {
+  readonly type = CLEAR_PRODUCT_PRICE;
+  constructor() {
+    super(PRODUCT_DETAIL_ENTITY, undefined, ProductScope.PRICE);
```

Comment:
What's the undefined field here?   Is int because we want to clear the price of all the products and not the price of a specific product?

Reply:
What is thee concept of `EntityScopedLoaderActions.EntityScopedResetAction` does it erase the data from the store, or does it refresh thee scope data with new calls to thee backend?

Reply:
yes, this `undefined` will clear all scoped data (in our case, prices of all products). It is designed as this.

Reply:
Then what mechanism reefreshes the data in this case?